README.md

```diff
@@ -68,8 +68,8 @@ Remove a selected book.
 
 #### 🔭 Future Features <a name="future-features"></a>
 
-- **Add a book**
-- **Remove a selected booke**
+- **Add Mobile vertion**
```

Comment:
- [ **OPTIONAL** ] I think it  would be  better if you correct the typing error to make your documentation looks more professional. I think it should be `Mobile version` instead of `Mobile vertion`